lychee-bin/tests/cli.rs

```diff
@@ -393,6 +393,42 @@ mod cli {
             .stdout(contains("3 OK"));
     }
 
+    #[test]
+    fn test_resolve_paths_from_root_path() {
+        let mut cmd = main_command();
+        let dir = fixtures_path().join("resolve_paths_from_root_path");
+
+        cmd.arg("--offline")
+            .arg("--include-fragments")
+            .arg("--root-path")
+            .arg(&dir)
+            .arg(dir.join("nested").join("index.html"))
+            .env_clear()
+            .assert()
+            .failure()
+            .stdout(contains("7 Total"))
+            .stdout(contains("5 OK"))
+            .stdout(contains("2 Errors"));
+    }
+
+    #[test]
+    fn test_resolve_paths_from_root_path_and_base_url() {
+        let mut cmd = main_command();
+        let dir = fixtures_path();
+
+        cmd.arg("--offline")
+            .arg("--root-path")
+            .arg("/resolve_paths")
+            .arg("--base")
+            .arg(&dir)
+            .arg(dir.join("resolve_paths").join("index.html"))
+            .env_clear()
+            .assert()
+            .success()
+            .stdout(contains("3 Total"))
+            .stdout(contains("3 OK"));
+    }
```

Comment:
copy of `test_resolve_paths` with splitting the base into two parts (base + root path)